### ISSUE TITLE
Allow writing a description for grouped time entries (like web app)

### ISSUE THREAD

**Issue by th0rgall** [toggl-open-source/toggldesktop#3987]
### 💻 Environment

Platform: macOS

### Current behavior on the *desktop* client (problematic)
1. Log 2 time entries without assigning them to a project/tag/...
2. They will get grouped
3. Click the group: the group will open
4. Change the title of each individual underlying time enry

### Expected behavior == current behavior on the *web* client
1. Log 2 time entries without assigning to a project
2. They will get grouped
3. Click the group: you can now write a description on the group. **The description will be filled to all underlying time entries.**

This also applies to *changing* the description of existing grouped time entries.

Basically I'd love to see this web client behavior on the desktop client too.

It seems #2004 and #2054 omitted this feature.

### ⭐️ Why do you want this?

Describing multiple entries at the same time saves me time. 

There is an annoying inconsistency now in features between web & desktop client.

**Comment by IndrekV** [comment]
The feature you are describing is group editing. We have an issue for that for some time #2059
We have not set down a clear deadline or plans to implement it. Thank you for bringing this up again and emphasizing it's importance.

**Comment by th0rgall** [comment]
Okay @IndrekV, thanks for your answer. I see, I'll close this duplicate then.
